### PR TITLE
Add Beneficiary, BeneficiaryParent and AgentDetail serializers and endpoints

### PR DESCRIPTION
**Pull request by jmnda-dev**
- [x] Add `Beneficiary` model serializer
- [x] Add `BeneficiaryParent` model serializer
- [x] Add `AgentDetail` model serializer
- [x] Add API endpoints for `Beneficiary`
- [x] Add API endpoints for `BeneficiaryParent`
- [x] Add API endpoints for `AgentDetail`

### CODE REVIEW

File: itez/beneficiary/models.py [hunk 1]
```diff
@@ -8,10 +8,10 @@ class AgentDetail(models.Model):
...
     Create agent detail table with its attributes or columns.
     """
-    MALE        =  1
-    FEMALE      =  2
-    TRANSGENDER =  3
-    OTHER       =  4
+    MALE        =  "1"
+    FEMALE      =  "2"
+    TRANSGENDER =  "3"
+    OTHER       =  "4"
```
Comment (sonlinux) [inline itez/beneficiary/models.py]: these values need to be integers

File: itez/beneficiary/api/views.py [hunk 2]
```diff
@@ -7,20 +9,74 @@
...
+    serializer_class = BeneficiaryParentSerializer
+
+    def get(self, request, *args, **kwargs):
+        return self.list(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        return self.create(request, *args, **kwargs)
+
+
+class BeneficiaryParentAPIView(ListModelMixin, CreateModelMixin,
```
Comment (sonlinux) [inline itez/beneficiary/api/views.py:68]: lets add some docs here

File: itez/beneficiary/api/views.py [hunk 3]
```diff
@@ -7,20 +9,74 @@
...
+    serializer_class = BeneficiarySerializer
+
+    def get(self, request, *args, **kwargs):
+        return self.list(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        return self.create(request, *args, **kwargs)
+
+
+class BeneficiaryParentAPIView(ListModelMixin, CreateModelMixin,
```
Comment (sonlinux) [inline itez/beneficiary/api/views.py]: lets add some docs here

File: itez/beneficiary/api/views.py [hunk 4]
```diff
@@ -7,20 +9,74 @@
...
+    serializer_class = AgentDetailSerializer
+
+    def get(self, request, *args, **kwargs):
+        return self.list(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        return self.create(request, *args, **kwargs)
+
+
+class BeneficiaryAPIView(ListModelMixin, CreateModelMixin,
```
Comment (sonlinux) [inline itez/beneficiary/api/views.py:50]: lets add some docs here

File: itez/beneficiary/api/views.py [hunk 5]
```diff
@@ -7,20 +9,74 @@
...
+    BeneficiarySerializer,
+    BeneficiaryParentSerializer,
     ProvinceModelSerializer,
     DistrictModelSerializer,
     ServiceAreaModelSerializer,
     WorkDetailModelSerializer
 )
 
 
+class AgentDetailAPIView(ListModelMixin, CreateModelMixin,
```
Comment (sonlinux) [inline itez/beneficiary/api/views.py:32]: lets add some docs here

**Review by sonlinux — CHANGES_REQUESTED** [review]

File: itez/beneficiary/api/views.py (hunk 2 above)
Comment (jmnda-dev) [reply itez/beneficiary/api/views.py:68]: Sorry about this, I forgot

File: itez/beneficiary/models.py (hunk 1 above)
Comment (sonlinux) [reply itez/beneficiary/models.py]: then after update migrations

**Review by sonlinux — APPROVED** [review]
LGTM
Thanks @jmnda-dev

**Pull request merged**